### DISCUSSION TITLE
http parameters when streaming remotely

### DISCUSSION THREAD

**Discussion by simondsmason** [blakeblackshear/frigate#15476] [Ask A Question]
Can someone point me at the webrtc Commandline parameters when streaming cameras remotely?  I have looked but haven't found it.

I have /stream.html?src=camera_name.  Is that it or are there more?

Thanks.

**Comment by hawkeye217** [comment]
This seems to be more of a go2rtc question rather than a Frigate one?

**Comment by simondsmason** [comment]
I found it.  If you go to the go2rtc dashboard there is an option called links for every camera with all of the possible ways to stream.  I haven't found this documented yet but the options are all nicely laid out here.
